libs/accounts/src/lib/accounts-data-provider.ts

```diff
@@ -50,3 +52,42 @@ export const accountsDataProvider = makeDataProvider<
   getData,
   getDelta,
 });
+
+export const getAccountData = (
+  data: AccountFieldsFragment[],
+  assetSymbol: string
+) => {
+  const assetData = data.filter((a) => a.asset.symbol === assetSymbol);
+
+  const deposited = assetData
+    .filter((a) => [AccountType.ACCOUNT_TYPE_GENERAL].includes(a.type))
```

Comment:
Should we count in deposited (GENERAL) or all available balance (i.e. plus REWARDS etc.)?

Reply:
I am calculating separately `deposited` and `available`.  I have categorized the accounts into `incoming` and `outcoming`.